src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -435,7 +446,8 @@ private void mpathwaysCall(HttpServletRequest request, HttpServletResponse respo
 				wapiValuesMap.put("secret", appExtSecurePropertiesFile.getProperty(SectionUtilityToolFilter.ESB_SECRET));
 				WAPI wapi = new WAPI(wapiValuesMap);
 				try {
-					wrappedResult = wapi.getRequest(wapi.getApiPrefix() + mpathwaysInstructor + "/Terms/" + mpathwaysTermId + "/Classes");
+					M_log.info("WAPI URL: " + wapi.getApiPrefix() + uniqname + "/Terms/" + mpathwaysTermId + "/Classes");
```

Comment:
Why not just assign the url to a string and use that string for the call and the log message.  The message can't get out of date then.